test/helpers/browser/driver.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Driver = require('../../../helpers/browser/driver.js');
+const Element = require('../../../helpers/element.js');
+const assert = require('assert');
+
+let DriverStub = new Driver();
+
+DriverStub.sendCommand = function(command, params) {
```

Comment:
yeah, this is great.

When we add gather tests, we'll need to add the other side of this, checking that requests to a browser returns objects with these structures.